README.md

```diff
@@ -33,7 +33,7 @@ inconveniences.
 Example using Homebrew on macOS:
 
 ```bash
-$ brew install llvm-hs/homebrew-llvm/llvm-5.0
```

Comment:
Is this correct? I don’t use homebrew so I honestly don’t know if replacing `homebrew-llvm` by `llvm` makes sense.

Reply:
Both versions would work: https://docs.brew.sh/Taps

>Repository naming conventions and assumptions
>On GitHub, your repository must be named homebrew-something in order to use the one-argument form of brew tap. The prefix ‘homebrew-‘ is not optional. (The two-argument form doesn’t have this limitation, but it forces you to give the full URL explicitly.)

>When you use brew tap on the command line, however, you can leave out the ‘homebrew-‘ prefix in commands.

But bash completion works only for version without prefix:

    brew install llvm-hs/llvm/llvm-

    llvm-hs/llvm/llvm-3.7  llvm-hs/llvm/llvm-3.8  llvm-hs/llvm/llvm-3.9  llvm-hs/llvm/llvm-4.0  llvm-hs/llvm/llvm-5.0  llvm-hs/llvm/llvm-6.0  

And I just successfully installed llvm-6 using the prefix-less command, so it works on my machine :)

Reply:
Alright, thanks!